catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceInputTransformer.java

```diff
@@ -256,8 +262,9 @@ private void parseRestrictions(MetacardImpl metacard, Object json)
   private Map<String, Object> getJsonObject(InputStream stream) throws CatalogTransformerException {
     String jsonString = null;
     try {
-      jsonString = IOUtils.toString(stream);
-      Map<String, Object> rootObject = MAPPER.parser().parseMap(jsonString);
+      jsonString = IOUtils.toString(stream, StandardCharsets.UTF_8);
```

Comment:
✏️ We were forced to read streams in a string with Boon because of a bug. Should we take advantage of Gson and read directly from it?

Reply:
We debug log and throw a potential exception that includes that json string. That's why I left this one as it was.